lib/templates/lab5/comment.dart

```diff
@@ -1,16 +1,36 @@
-// TODO add dependencies
-// TODO add comment.g.dart as part
+import 'package:dio/dio.dart';
+import 'package:json_annotation/json_annotation.dart';
 
+part 'comment.g.dart';
+
+@JsonSerializable()
 class Comment {
   // TODO task 2 to make this class for url http://jsonplaceholder.typicode.com/comments
```

Comment:
You have only 2 tasks completed

Reply:
Yes, I've spent too much time to fix some errors with flutter (`dart run build_runner build` wasn't working)